scripts/Dockerfile

```diff
@@ -11,8 +11,4 @@ RUN apt-get update && \
 # dxCompiler
 COPY dxCompiler-$VERSION.jar /dxCompiler.jar
 
-# Copy the entrypoint script
```

Comment:
The entrypoint script should still be kept for this one, though

Reply:
sorry, just mixed up when reverting.